src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action in actions) {
```

Comment:
don't use `for in` to iterate through array, you have `for of` for that
```suggestion
  for (const action of actions) {
```

Reply:
Mykola told you to use 'for in' in 11 line, you should use 'for of' when you iterate over actions